nfc/CHANGELOG.rst

```diff
@@ -34,6 +47,8 @@ Added
 * A dedicated header file :file:`nrf_nfc_errno.h` with error codes.
 * A source file that translates nrfx error codes to NFC library error codes.
 * Definitions for returned error codes in the API documentation.
+* Intermediate callback processing with :c:func:`nfc_platform_cb_request` function.
+* A header :file:`nfc_platform.h` file for platform layer.
```

Comment:
This note is in the `nRF Connect SDK v1.9.0 / Added` section. 
Is this correct?  The `nfc_platform.h` file is added in this PR.